### ISSUE TITLE
Authorization Manager - Failed ( microsoft/dynamics-nav:generic)

### ISSUE THREAD

**Issue by NAVspecialist** [Microsoft/nav-docker#132]
Evening

I am trying to do a:

docker run -e accept_eula=Y -v C:\Users\Palle\Desktop\NAV2015u38:c:\navdvd microsoft/dynamicsnav:generic
The installation seems to start but unfortunately I am getting an error:

Installing NAV
Starting Local SQL Server
Starting Internet Information Server
Installing Url Rewrite
Installing Report Viewer
Installing OpenXML
Copying Service Tier Files
Copying Web Client Files
Copying Windows Client Files
Copying PowerShell Scripts
Copying UpgradeToolKit
Copying ClientUserSettings
Copying ReportBuilder
**AuthorizationManager check failed.**

Any Idea what could be wrong?

Thanks
Palle

**Comment by freddydk** [comment]
What is the exact docker run command?
Long shot: It might be because the folder with the NAVDVD is on your desktop, which might have permissions.

**Comment by NAVspecialist** [comment]
docker run -e accept_eula=Y -v C:\nav2015:c:\navdvd microsoft/dynamicsnav:generic

Unfortunately was the problem not solved by unpacking the DVD to a new folder.

I have tried upper/lowercase foldername (just to see if it changed anything - it didn't),

**Comment by freddydk** [comment]
There is nothing called microsoft/dynamicsnav:generic
it is called microsoft/dynamics-nav:generic, which I assume you must be using, else you wouldn't get that far. Did you try to follow some of the troubleshooting items on my blog to see if you can run basic WindowsServerCore containers?
Thanks

**Comment by Koubek** [comment]
Just for curiosity, the same error message (**AuthorizationManager check failed**) can be seen here #128 as well.

**Comment by NAVspecialist** [comment]
The missing dash (-) was a copy/paste issue from my commandline. It's there!
I am running the the command elevated as administrator.

I have tried your troubleshooting guide: no 1, 3, 4, 7 with no problems. WIndowsServerCore works quite fine.
I have tried NAV2015 update (37,35,34,30)
Furthermore I have tried NAV 2017 Update 13 in either case the Docker log says:

Installing NAV
Starting Local SQL Server
Starting Internet Information Server
Installing Url Rewrite
Installing Report Viewer
Installing OpenXML
Copying Service Tier Files
Copying Web Client Files
Copying Windows Client Files
Copying PowerShell Scripts
Copying UpgradeToolKit
Copying ClientUserSettings
Copying ReportBuilder
AuthorizationManager check failed.

**Comment by freddydk** [comment]
Are you running Windows 10?
have you tried to insert -m 4G to give the container 4GB memory?

**Comment by NAVspecialist** [comment]
Yes! -m 4G same error its only the generic image that giving me the troubles.

Just tried:

docker pull microsoft/dynamics-nav:2017-cu13-dk
docker run --rm --name nav2017u13 -h nav2017u13 -e accept_eula=Y -m 4G -e clickonce=Y -e usessl=N -e username="admin" -e password="Passw0rd" microsoft/dynamics-nav:2017-cu13-dk 
which works nicely.

**Comment by freddydk** [comment]
I just tried the generic which works fine here.
Any chance that you could try another machine?

**Comment by Koubek** [comment]
Is the machine domain-connected?

**Comment by freddydk** [comment]
http://tgnp.me/2011/09/powershell-authorizationmanager-check-failed-resolution/

– Occasionally, with the ExecutionPolicy set to “RemoteSigned” or “Unrestricted”, this error may occur if the script was downloaded from the internet using Internet Explorer and IE has “blocked” the file. Right-click the file, select “Properties” and click “Unblock”.

Did you unblock the NAVDVD zip file?

**Comment by NAVspecialist** [comment]
YES!!! 

As Freddy 🥇  pointed out, the image zip-files is blocked before it was unzipped. 

What I did was: unblocked the file, unzipped the filed, unzipped the DVD to C:\NAV2015 (and C:\NAV2017) and then I tried to do a:

docker run -e accept_eula=Y -e usessl=N -m 4G -v C:\NAV2015:c:\navdvd microsoft/dynamics-nav:generic (and C:\NAV2017)

They both works!!! THANKS a LOT!!! 
The spin-up took merely 103 seconds. I am amazed!

And yes in order to do a: install-module navcontainerhelper -force in powershell I had to do a

Set-ExecutionPolicy RemoteSigned (also in powershell)  

which seems to have gotten me into this mess.

Have a great evening 

Palle
